### PR TITLE
Amazon sentiment analysis

### PR DESCRIPTION
**Pull request by Raghvender1205**
### Description
This is a NLP Sentiment Analysis on Tensorflow Datasets Amazon Review data. I have used DistilBERT pretrained for this.

I have a screenshot of the test sentence.
Output

I have gotten 92+% accuracy.
Accuracy

### CODE REVIEW

**Comment by Raghvender1205** [comment]
Can you please tell me why you closed the pull request maybe I can alter something and again apply for a pull request?

**Comment by Raghvender1205** [comment]
So what you want me to do? I want to submit the PR so should I add readme
file with the notebook?
